Frends.Jira.UpdateIssue/Frends.Jira.UpdateIssue/Definitions/Connection.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.Jira.UpdateIssue.Definitions;
+
+/// <summary>
+/// Connection parameters.
+/// </summary>
+public class Connection
+{
+    /// <summary>
+    /// Jira base URL.
+    /// </summary>
+    /// <example>https://test.jira.fi/jira</example>
+    public string JiraBaseUrl { get; set; }
+
+    /// <summary>
+    /// Bearer token.
```

Comment:
Maybe the description should indicate that user has to provide Personal Access Token of a Jira user.